src/Workspaces/Core/Portable/project.json

```diff
@@ -1,8 +1,24 @@
 ﻿{
-  "dependencies": {
-    "Newtonsoft.Json": "6.0.4"
-  },
-  "frameworks": {
-    ".NETPortable,Version=v4.5,Profile=Profile7": {}
-  }
+    "supports": {
+        "net46.app": {}
+    },
+    "dependencies": {
+        "Microsoft.NETCore.Portable.Compatibility": "1.0.1",
+        "Newtonsoft.Json": "6.0.4",
+        "System.Threading.Tasks": "4.0.11",
```

Comment:
@CyrusNajmabadi, `NetStandard.Library` should be `1.6.0`, you also need to add the `Microsoft.NETCore.Portable.Compatibility` package (`1.0.1`). There are also some of the packages that are not included in the NetStandard.Library and still need to be explicitly referenced (I think `System.Diagnostics.Contracts` is one of them.